src/gatsby/src/styles/search.scss

```
@@ -310,19 +310,27 @@ div[class$="-singleValue"] {
 }
 
 .park-heading-text {
-  display: -webkit-box;
-  -webkit-line-clamp: 2;
-  -webkit-box-orient: vertical;  
-  overflow: hidden;
+  display: inline-flex;
+  position: relative;
```

If we want to center the text and the icon vertically, we can add `align-items: center;` to the container - then we can remove the `position: relative;` and all of the styles on the `.park-heading-icon` except the (font size).
```suggestion
  display: inline-flex;
  align-items: center;
  gap: 0.5em; // some spacing to keep the distance between the icon and the text - not sure how much
```

I think the result will be the same, but more robust because flex box will handle all the positioning.

---

Thanks for your suggestion, works like a charm 👏